File: engine/model/src/main/scala/net/server/channel/packet/cash/operation/IncreaseInventorySlotsSmall.scala
package net.server.channel.packet.cash.operation

class IncreaseInventorySlotsSmall(private var _action: Int, private var _cash: Int, private var _mode: Byte, private var _type: Byte) extends BaseCashOperationPacket(_action = _action) {
  def cash: Int = _cash

  def mode: Byte = _mode

  def theType: Byte = _type
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/IncreaseInventorySlotsSmall.java
package net.server.channel.packet.cash.operation;

public class IncreaseInventorySlotsSmall extends BaseCashOperationPacket {
   private final Integer cash;

   private final Byte mode;

   private final Byte theType;

   public IncreaseInventorySlotsSmall(Integer action, Integer cash, Byte mode, Byte theType) {
      super(action);
      this.cash = cash;
      this.mode = mode;
      this.theType = theType;
   }

   public Integer cash() {
      return cash;
   }

   public Byte mode() {
      return mode;
   }

   public Byte theType() {
      return theType;
   }
}
